x-pack/plugins/ingest_manager/server/services/agents/acks.test.ts

```diff
@@ -50,21 +61,19 @@ describe('test agent acks services', () => {
 
   it('should fail for actions that cannot be found on agent actions list', async () => {
     const mockSavedObjectsClient = savedObjectsClientMock.create();
+
+    mockSavedObjectsClient.bulkGet.mockReturnValue(
+      Promise.resolve({
+        saved_objects: [],
+      } as SavedObjectsBulkResponse<any>)
```

Comment:
Is it possibly to wrap the any in a type. I am assuming you expect this to be some type of index type?